### DISCUSSION TITLE
可以成功启动但是youtube的访问检查连接超时

### DISCUSSION THREAD

**Discussion by cironfish** [vernesong/OpenClash#2565] [Q&A] [answered]
手机端的ytb客户端可以刷出来时间线，但是打不开视频，也用不了谷歌之类的。
查了一下可能是ipv6的问题，之前启动时也有ipv6相关的警告，按照网上的教程把ipv6都关了以后警告消失了，但还是连不上谷歌之类的。
自己查了一天也没有彻底解决，没办法只能来这边请教一下了，谢谢。

补：ip地址那边也是前三个都是国内地址，第四个IPIFY    国外是查询中，但之前偶尔第三个地址也有显示过国外的ip。

睡了一觉起来能打开ytb了，里面的视频有些能看有些看不了，但是谷歌还是打不开。

Openclahs版本：0.45.35-beta
OpenWrt：R22.3.13 (2022-07-23) / LuCI Master (git-22.190.06436-4a8cc2a)

**Comment by vernesong** [comment, marked as answer]
DNS的问题吧，你关闭其他插件，打开劫持和fakeip模式，客户端查询dns看是不是fakeip

**Reply by cironfish** [reply]
谢谢回复。

这样设置之后插件里显示github和ytb连接正常，但是用浏览器访问只能打开页面看不了视频，也打不开谷歌。

**Reply by Snailya** [reply]
同样的问题，友善r2s做旁路由，fakei-ip模式IP地址一直查询中，网站访问全是无法访问。
1. 手机wifi设置代理指向旁路由http端口，可以访问谷歌，youtube，但无法加载视频。
2. 手机wifi设置静态ip，并指定网关和dns为旁路由，无法连接网络。

**Reply by eddieY1** [reply]
就这样设置之后，关闭服务再开启，等两分钟，解决了，感谢

**Comment by boyzhaojian** [comment]
1.我跟你用的相同的固件版本，也是7-23这天编译的版本，用到现在非常稳定。我想备份一份固件，但是我忘记是在哪下载的了。请问你还记得这个固件是谁家的吗？

**Comment by JaysonPeng** [comment]
确实太难用了，换passwall了
